### PR TITLE
Add "all" as an option to restart all deployments

### PR DESCRIPTION
**Pull request by kirs**
I'm working on adding support for restarting deployments from the Shipit UI, and I think it would be useful to have an option to restart all of them (for instance web, jobs and redis).

### CODE REVIEW

File: lib/kubernetes-deploy/restart_task.rb [hunk 1]
```diff
@@ -45,6 +45,11 @@ def perform(deployments_names)
         raise ArgumentError, "#perform takes at least one deployment to restart"
       end
 
+      if deployments_names == "all"
```
Comment (kirs) [inline lib/kubernetes-deploy/restart_task.rb]: Shipit has no data about what deployments in the app so kubernetes-deploy would have to query the list of all deployments first.

**Comment by ibawt** [comment]
redis very dangerous to restart.  ES as well, there's a few of them.

Can we do this for deployments with containers that match the repo name? wdyt

**Comment by kirs** [comment]
@ibawt here is the use case: the app owner wants to restart in Shipit:

screen shot 2017-04-24 at 17 03 39

But they don't know anything about deployments. They just want to restart it. My idea was to provide two generic resources (web and jobs) and "all" as an option just in case.

I'm open to suggestions what would be the best experience for developers. I'm not sure how repo name matching is relevant here.

**Comment by ibawt** [comment]
you can filter the deployments by the predicate of their image matching the code they ship.

**Comment by kirs** [comment]
Implemented in https://github.com/Shopify/kubernetes-deploy/issues/90

**Pull request closed without merging**